### ISSUE TITLE
add "how to add HTTPS to your services" getting started guide

### ISSUE THREAD

**Issue by dovholuknf** [openziti/ziti-doc#251]
see https://openziti.discourse.group/t/using-https-to-access-services-over-openziti/901

User mentioned HTTPS - it'd be great to point people how to acquire certificates from a verified domain, and then how to use __ totally private DNS__ provided by openziti to have "valid" certificates. Specifically, using the keypair for services provided by OpenZiti, not the OpenZiti overlay

**Comment by sadathknorket** [comment]
+1

**Comment by dovholuknf** [comment]
possibly also relates to https://github.com/openziti/ziti-doc/issues/84 seems like they are dupes to me perhaps

**Comment by poppaLol** [comment]
Its possible its duplicated, but there is a little extra nuance here.

When I was reading the discourse conversation I took this to mean how to present an endpoint on a DNS name, which is inside the fabric, and link up the certificates on the hosted service behind the tunnelled bind policy.

I managed to make this work for my set up. Its nice for my use case, because it keeps a developer instance of an SPA I am writing secret and only accessible to a few people. I thought it might make a nice article, and its more about serving certificates in bespoke SPA served via ZT fabric.

I dont mind putting a quick write up together, even if we decide collectively to drop it later

**Comment by dovholuknf** [comment]
ah yes I think I understand. so this issue turns into "https on **your** services" right? If that's what you're thinking that's cool and is definitely not the same as the other issue which is about using LE certs on your overlay (not your individual services).

Assuming I understand you, I'll update this issue's title to more acccurately reflect

**Comment by poppaLol** [comment]
Yes, we are on the same page! Seems to make sense to put this somewhere close to "quickstarts - services - your first service", but let me know if you want place the info elsewhere.

**Comment by dovholuknf** [comment]
sounds great - i look forward to it. i'll update the issue and description to make it a bit more clear :) thanks

**Comment by dovholuknf** [comment]
Just a quick "general vibe guideline" comment:

This should be located in our "guides" section and it is acceptable to use words like "you"/"your" etc. It should be direcrted to the reader as though you are speaking to them and it should guide them directly through with a "if you do all these things the way this guide shows you at the end you'll have HTTPS services". It doesn't need to cover all options but it should help a new person coming to the project understand what they are about to do, why, and how.

Thanks again for taking this up. :)

**Comment by poppaLol** [comment]
Got it! Thanks for the direction!
